sopel/plugins/handlers.py

```diff
@@ -618,8 +618,9 @@ def get_version(self) -> Optional[str]:
         ):
             try:
                 version = importlib.metadata.version(self.module.__package__)
-            except ValueError:
-                # package name is probably empty-string; just give up
+            except:
```

Comment:
Linter won't be happy unless you use:

```suggestion
            except Exception:
```